predicators/nsrt_learning/strips_learning/base_strips_learner.py

```diff
@@ -43,14 +43,43 @@ def learn(self) -> List[PartialNSRTAndDatastore]:
         filtering may break it.
         """
         learned_pnads = self._learn()
+        # Remove pnads by increasing min_data_perc until harmlessness breaks.
```

Comment:
cool! I didn't know that this was how you implemented this specifically. I like it